include/bouncer.h

```diff
@@ -369,6 +369,13 @@ struct PgDatabase {
  */
 struct PgSocket {
 	uint32_t client_id; /* A "unique" id of the client */
+	struct IncompletePacketBuffer {
```

Comment:
small nit -- probably better to define this as a type within our logging `.h` file